### PR TITLE
Migrate OracleJDK to OpenJDK in images

### PR DESCRIPTION
**Pull request by mshaposhnik**
To improve licensing purity, predefined agent images now switched to OpenJDK.

### CODE REVIEW

**Comment by mshaposhnik** [comment]
@eivantsov @riuvshin pls review

File: recipes/debian_jdk8/Dockerfile [hunk 1]
```diff
@@ -24,22 +25,14 @@ LABEL che:server:8080:ref=tomcat8 che:server:8080:protocol=http che:server:8000:
 
 
 ENV MAVEN_VERSION=3.3.9 \
-    JAVA_VERSION=8u45 \
-    JAVA_VERSION_PREFIX=1.8.0_45 \
+    JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64 \
```
Comment (mshaposhnik) [inline recipes/debian_jdk8/Dockerfile]: Can we avoid hardcoding of this?

**Comment by riuvshin** [comment]
maybe we have to use official openJDK image? https://hub.docker.com/_/openjdk/

**Comment by skabashnyuk** [comment]
@TylerJewell  WDYT about @riuvshin 's proposal?

**Comment by eivantsov** [comment]
Official image is based on either debian or alpine. Many eclipse che images use Ubuntu as a base image. I don't mind using alpine, but it means we have to rebuild ALL images that we have

**Comment by riuvshin** [comment]
ahhhh Im sorry this is stack images! I thought this is  image where we run che or codenvy, I don't care about this one.

**Comment by skabashnyuk** [comment]
our base image eclipse/openjdk8 we can inherit from debian based 8u111-jdk image. alpine is not good for development.

**Comment by eivantsov** [comment]
Yes, but again that is more work to be done - basically redo ALL images and test them.

**Comment by skabashnyuk** [comment]
Yes but in this way we avoid any headache with jre or jdk licensing.

**Comment by TylerJewell** [comment]
@eivantsov - none of our enterprise customers has made an issue yet of our stack images using Oracle JDK, but eventually they will.  right now we would tell them that they need to build their own stacks that use different JVMs.  But I think eventually they will ask us to have a version that doesn't require license acceptance, ... or what they will do is ask us to inject their corporate Oracle JVM license as part of the workspace runtime start (instead of auto-accepting the license that is included).

**Pull request merged**